src/vaadin-dialog-draggable-mixin.html

```diff
@@ -0,0 +1,47 @@
+<script>
+  /**
+   * @namespace Vaadin
+   */
+  window.Vaadin = window.Vaadin || {};
+
+  /**
+   * @polymerMixin
+   * @memberof Vaadin
+   */
+  Vaadin.DialogDraggableMixin = subclass => class VaadinDialogDraggableMixin extends subclass {
+    ready() {
+      super.ready();
+      this._addDraggableEventListeners();
+    }
+
+    _addDraggableEventListeners() {
+      let originalBounds;
+      let originalMouse;
+      this._stopDrag = () => {
+        window.removeEventListener('mouseup', this._stopDrag);
+        window.removeEventListener('mousemove', this._drag);
+      };
+      this._drag = (e) => {
+        const top = originalBounds.top + (e.pageY - originalMouse.top);
+        const left = originalBounds.left + (e.pageX - originalMouse.left);
+        this._setBounds({top, left});
```

Comment:
Should probably be wrapped in something like `if (e.clientX >= 0 && e.clientX <= window.innerWidth && e.clientY >= 0 && e.clientY <= window.innerHeight) {`. If you drag the window beyond the leftmost border of the viewport, you can no longer access it without a browser refresh. Resizing should probably have the same check. V8 seems to work the same https://demo.vaadin.com/sampler/#ui/structure/window

Reply:
Done.